sdcm/cluster.py

```diff
@@ -1990,6 +1990,7 @@ def install_scylla(self, scylla_repo):
                 install_debian_10_prereqs = dedent("""
                     export DEBIAN_FRONTEND=noninteractive
                     apt-get update
+                    rm /etc/apt/trusted.gpg || echo "could not delete /etc/apt/trusted.gpg"
```

Comment:
may be 'rm -f

Reply:
could be, but that will hide the error in case the file is busy or doesn't exist